src/util/db/__jest__/IgnoreList.test.ts

```diff
@@ -0,0 +1,15 @@
+import { add, remove, exists } from '../IgnoreList';
```

Comment:
This test will write directly into `db.json` due to how `connection` is set up. Usually that is not what we want... Imagine something going wrong, while the test is running, the test data would end up in `db.json` :sob:

I can think of two solutions. Either we mock the whole `connection.ts` to work on a different file, or we mock all calls of lowdb to read the value of the file contents. Don't knwo what would be more idiomatic, whatever you prefer. I guess, the first one would be a lot easier.

Could you make that happen?

Reply:
Good point.  I can find a way to change the file location to one that is obviously a test file.

I don't think mocking the lowdb calls would be a good idea, since this bug was because of a incorrect assumption about how lowdb works in the first place!